topology/sof-whl-hda.m4

```diff
@@ -27,99 +27,99 @@ include(`platform/intel/bxt.m4')
 # PCM5  ----> passthrough (pipe 9)   -----> iDisp3 (HDMI/DP playback, BE link 5)
 #
 
-# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s32le.
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s16le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-passthrough-playback.m4,
-	1, 0, 2, s32le,
+	1, 0, 2, s16le,
```

Comment:
we should really have a volume control for all HDMI outputs (this will also convert 32 to 24bit - dont use 16 when 24 is available)

Reply:
ok. I will talk with CCG team whether they need a volume component.
And if they need, sof-hda-generic.m4 is ready for them

Reply:
I would reserve this PR, because it is in release note for customer.
And I would send another PR for WHL after discussing with customer.